src/download.jl

```diff
@@ -0,0 +1,51 @@
+"""
+dataurl(dataset_code)
+This function takes in a dataset code of format "GADM/<country_code>"
+Returns the URL of the gpkg dataset of the country
+"""
+function dataurl(dataset_code)
+
+    # Splits GADM/IND to GADM, IND
+    dataset_provider, country_code = split(dataset_code, "/")
+    
+    # Only accepts GADM Dataset Provider
+    if dataset_provider != "GADM"
+        error("❌ Dataset Provider $dataset_provider not supported. 💡 Please try \"GADM/<Code>\".")
```

Comment:
Now that we have proper logging decoration, I think it looks more professional if we remove the unicode symbols. Also, please use `@error` for consistency with `@warn` and `@info`.